uint8 command
uint8 id
uint8[4] data

